### PR TITLE
15.0 mig delivery carrier category

### PR DESCRIPTION
**Pull request by pere-aquarian**

### CODE REVIEW

File: delivery_carrier_category/security/delivery_carrier_category.xml [hunk 1]
```diff
@@ -3,50 +3,50 @@
      License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
 <odoo>
 
-    <record model="ir.model.access" id="delivery_carrier_category_salesman">
+    <record model="ir.rule" id="delivery_carrier_category_salesman">
```
Comment (rousseldenis) [inline delivery_carrier_category/security/delivery_carrier_category.xml]: @pere-aquarian You cannot do that!

Please restore ir.model.access model.

See: 
image

**Review by rousseldenis — CHANGES_REQUESTED** [review]
@pere-aquarian Thanks for this.

Could you attend comment and as module exists in v14, migrate from that one?

https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-15.0

File: delivery_carrier_category/security/delivery_carrier_category.xml (hunk 1 above)
Comment (pere-aquarian) [reply delivery_carrier_category/security/delivery_carrier_category.xml]: done! @rousseldenis

File: delivery_carrier_category/tests/test_carrier_category.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
+# Copyright 2020 ACSONE SA/NV (<http://acsone.eu>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from psycopg2 import IntegrityError
+
+from odoo.tests.common import SavepointCase
+from odoo.tools import mute_logger
+
+
+class TestCarrierCategory(SavepointCase):
```
Comment (rousseldenis) [inline delivery_carrier_category/tests/test_carrier_category.py]:
```suggestion
class TestCarrierCategory(TransactionCase):
```

**Review by rousseldenis — CHANGES_REQUESTED** [review]
Could you change test class to TransactionCase and squash your commits to keep migration one ? Thanks

File: delivery_carrier_category/tests/test_carrier_category.py (hunk 2 above)
Comment (pere-aquarian) [reply delivery_carrier_category/tests/test_carrier_category.py]: Done!

**Comment by pere-aquarian** [comment]
Thank you for the patience, @rousseldenis . Just a rookie on that and I am starting from the easiest modules to understand the whole process. Think I am done. Commits squashed and only one left with the migration one.

**Comment by rousseldenis** [comment]
> Thank you for the patience, @rousseldenis . Just a rookie on that and I am starting from the easiest modules to understand the whole process. Think I am done. Commits squashed and only one left with the migration one.

@pere-aquarian No problem, always a pleasure to welcome new people contributing ! I'll try to look at it today

**Comment by pere-aquarian** [comment]
@jcbr87 Could you please take a look at it? 
Thank you!

**Comment by pere-aquarian** [comment]
Hi, @rousseldenis !

Is there something more I have to do? Sorry my ignorance.

Thank you!

**Comment by rousseldenis** [comment]
@pere-aquarian It seems your commit history is quite a mess as there is another module in it.

Did you follow the migration guide I've provided here above (especially the Full process part) ?

https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-15.0

**Pull request closed without merging**